### PR TITLE
Update beta text in mapviewer.

### PR DESCRIPTION
**Pull request by alan-wu**
# Description

Update beta text display in Map viewer based on the following feedbacks. Hyperlinks have been added to some of the text.

## Type of change

Delete those that don't apply.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

I have tested this on locally deployed instance.

It can be tested here - https://alan-wu-sparc-app.herokuapp.com/

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -11,10 +11,10 @@
     orbit-camera-controller "^4.0.0"
     turntable-camera-controller "^3.0.0"
 
-"@abi-software/flatmap-viewer@^2.3.3-b.4":
-  version "2.3.3-b.4"
```
Comment (hsorby) [inline yarn.lock:15]: How has one version number gone backwards and another gone forwards?

**Review by Tehsurfer — APPROVED** [review]
Looks good, am interested in Hughs comment though

File: yarn.lock (hunk 1 above)
Comment (alan-wu) [reply yarn.lock:15]: Looks like it is the automated increment when installing the package on mapintegrated viewer.

**Comment by alan-wu** [comment]
> Looks good, am interested in Hughs comment though

Automated version bump by npm install on map viewer. All features are still working intentionally I believe.

**Pull request merged**